### DISCUSSION TITLE
FAQ page update budget

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#4] [Rewards]
The website's [FAQ page](https://www.asastats.com/faq/) has been updated! 

I want to thank everybody who has been involved, especially @SCN9A who made that [semi-final](https://github.com/asastats/static-pages/issues/2) step. :)

And that brings us to a problem that was one of the major reasons for abandoning the Google Docs platform as evaluating work there is a mission impossible.

I'm opening a discussion for defining a rewards budget for this FAQ page update. I guess we can define both the budget for the work that has already been done and for the future changes.

**Comment by AlgoRhythMatic** [comment]
Based on our most recent method of rewards calculation per COC/PonziCream method - we established the following guidance for calculating content generation tasks:
CT1 - The task takes up to 1 hour to complete: .01 damo
CT2 - The task takes 1 to 2 hours to complete: .02 damo
CT3 - The task takes 4 or more hours to complete: .04 damo

When looking at this directly now, note that it is currently valuing content editor's time at ~3.00 USD per hour, which is quite low.  I do like the C1/2/3 levels for guidance, but I think the values should be adjusted.

**Reply by ipaleka** [reply]
Yes, please adjust the values, both Algo and ASASTATS have decreased by around 250% in the last months and those two multiplies. I guess you can increase the general coefficient for all the rewards at once, right?

**Reply by AlgoRhythMatic** [reply]
Edit: I also meant to note that I think the following folks should then join the discussion to list how many hours they spent working on the Google docs portion to understand how best to calculate:
- Sammyspeed
- bear1bear2bear3
- Babbex22

**Reply by AlgoRhythMatic** [reply]
I see we also have a Github "poll" discussion type option, so maybe we could use this to ask the community what they feel is fair in terms of coefficient to use to increase rewards?  I think I can add ~4 options and let people vote to choose.

**Reply by Sammyspeed75** [reply]
We (@AlgoRhythMatic, Babbexx, and myself) spent a couple of weeks trying to finetune the FAQs doc before the final work by @SCN9A.

**Reply by ipaleka** [reply]
> We (@AlgoRhythMatic, Babbexx, and myself) spent a couple of weeks trying to finetune the FAQs doc before the final work by @SCN9A.

Thanks for replying.

I'm afraid those things aren't measured by substracting the end date with the start date.

You guys can either bring the number of hours each of you **actively** spent on working on that task or we can go other way around and discuss how much the whole thing adds to the project and your share in that.

**Reply by ipaleka** [reply]
Btw, my *special thank* to SCN9A is because those two pages would have stayed forever there in limb if he hadn't taken the move.

**Reply by AlgoRhythMatic** [reply]
> We (@AlgoRhythMatic, Babbexx, and myself) spent a couple of weeks trying to finetune the FAQs doc before the final work by @SCN9A.

Sammyspeed75 - please take a look at this recent discussion on [rewards review](https://github.com/asastats/channel/discussions/44).  For the FAQ task, the draft I proposed handled this one in a unique way, as I know a lot several people spent a bit of time on it.  I counted your initial work as 100% of a [C3], as I remember the effort you put in.  I also classified the same 100% @ [C3] for SCN9A for finalizing and delivering it.  I had Bear and I as editors at 50%, and missed including Babbexx22  (I'll add him).  IMO, there was a lot of community engagement on this one - I think it just arrived at a very busy time when folks were also very focused on COC and rewards process, which may account for it stalling.

Edit:
Here is what I now have for all contributors of this task - please advise if you think it is fair or not:
Type/Impact - [C3]
Sammyspeed - 100% (initiator/contributor)
Babbexx - 50% (contributor)
AlgoRhythMatic - 25% (editor)
Bear1bear2bear3 - 25% (editor)
SCN9A - 100% (contributor/editor/delivery)

As noted, usually total % should = 100, but there was a LOT of time spent on this one, and I think in good faith it is worth the exception.

**Reply by ipaleka** [reply]
Great and precise comment as always AlgoRhythm, thank you so much!

> I also classified the same 100% @ [C3] for SCN9A for finalizing and delivering it.

I don't want to affect any part of the process, let alone doubt of **your** reasoning. On top of that, I really don't have any insight on anyone's contribution in this task.

Right under this quote I just want to emphasize something else: enthusiasm and initiative should be rewarded separately from the related task. When we discuss members contributions in a particular thread for a particular task then only the work done should be taken in account.

In this example that means that SCN9A **initiative** taken (that *special thank*) should not affect his rewards for this task - tho, again, I'm not saying it did as I lack any insight. A matter of fact is that he has been rewarded among other things for that. So far I have been the person who has decided about that, but eventually the community should take main role in that too.

**Reply by wxivd** [reply]
@AlgoRhythMatic @ipaleka I am really happy to be excluded from FAQ rewards. Rewarding 300% in total for one page is a lot of resources. I appreciate the amount of work you guys have done for creating the substance which I relied on. Therefore, please remove SCN9A - 100% (contributor/editor/delivery). I feel more comfortable that way,  hence I have been rewarded for enthusiasm and initiative. I believe there are a lot of tasks to work on and rewards should not take more time to discuss.

**Reply by ipaleka** [reply]
> @AlgoRhythMatic @ipaleka I am really happy to be excluded from FAQ rewards.

And that's contrary to what we want to achieve here.

> Rewarding 300% in total for one page is a lot of resources. 

If AlgoRhythm thinks that's justified then I fully support his decision on this, if nothing else then because he has been pretty much the only person with concentration and insight on all the community rewards in 2022.

> I appreciate the amount of work you guys have done for creating the substance which I relied on. Therefore, please remove SCN9A - 100% (contributor/editor/delivery). I feel more comfortable that way, hence I have been rewarded for enthusiasm and initiative.

Please, our only goal is to define relative and absolute contributions to a particular task. There's no perfect insight on this task - that's exactly the reason why we have abandoned Google Docs as our platform - so everybody of us has to contribute in defining those (at least) relative values.

Your suggestion is an obstacle in the process as rewarding enthusiasm and initiative should be a policy in ASA Stats that will motivate community members. If we start to count the rewards members have got for that against the rewards they have deserved for the work done then the whole thing would undoubtedly become counterproductive.

I appreciate modesty, but it's actually an obstacle here. 

What we need is that you try to comprehend yourself as somebody else, preferably someone who you like and respect, and then define a relative work and share of that *other person* in total work.

> I believe there are a lot of tasks to work on and rewards should not take more time to discuss.

That's true, but if we can't define rewards as a community then we would revert back to the situation where I define them. And then I'd decide that there would be no more rewards in ASA Stats as there can't exist my counterpart when the DAO takes over. Simple as that.

P.S. I am aware that you wrote "should not take **more** time to discuss", so my whole last paragraph is a strawman fallacy, but let it stay as is.

**Reply by AlgoRhythMatic** [reply]
I 100% agree with @ipaleka on this one!  This was a very unique situation, and an exercise to show us what does and does not work - from a Tasker-perspective (at the time) as well as from Community engagement perspective.

I believe the dialog here is actually EXACTLY what we want from a DAO-perspective, talking it all through out in the open so that we all understand one another.  

My goal with the strange 300% was just to properly credit all involved, as I was close to this one in various chat threads, and feel I understand the efforts that all put into it.  One of the unseen benefits here is that some of the difficulties we ran into helped inform the Tasker COC + rewards process.  It also clearly showed us how difficult it is completely "crowd-source" content with a lot of authors, and how it can stall.

In any case, I really appreciate how humble you are about all of this SCN9A, and hope we continue to see your passion here, as I know it will motivate others.

**Reply by Sammyspeed75** [reply]
> > We (@AlgoRhythMatic, Babbexx, and myself) spent a couple of weeks trying to finetune the FAQs doc before the final work by @SCN9A.
> 
> Sammyspeed75 - please take a look at this recent discussion on [rewards review](https://github.com/asastats/channel/discussions/44). For the FAQ task, the draft I proposed handled this one in a unique way, as I know a lot several people spent a bit of time on it. I counted your initial work as 100% of a [C3], as I remember the effort you put in. I also classified the same 100% @ [C3] for SCN9A for finalizing and delivering it. I had Bear and I as editors at 50%, and missed including Babbexx22 (I'll add him). IMO, there was a lot of community engagement on this one - I think it just arrived at a very busy time when folks were also very focused on COC and rewards process, which may account for it stalling.
> 
> Edit: Here is what I now have for all contributors of this task - please advise if you think it is fair or not: Type/Impact - [C3] Sammyspeed - 100% (initiator/contributor) Babbexx - 50% (contributor) AlgoRhythMatic - 25% (editor) Bear1bear2bear3 - 25% (editor) SCN9A - 100% (contributor/editor/delivery)
> 
> As noted, usually total % should = 100, but there was a LOT of time spent on this one, and I think in good faith it is worth the exception.

Done that. thanks
